roles/servicetelemetry/defaults/main.yml

```diff
@@ -46,6 +46,12 @@ servicetelemetry_defaults:
     events:
       elasticsearch:
         enabled: false
+        extHostUrl: https://elasticsearch-es-http:9200
+        extTlsServerName: elasticsearch-es-http.{{ ansible_operator_meta.namespace }}.svc.cluster.local
+        extTlsSecretName: elasticsearch-es-cert
+        extUserSecretName: elasticsearch-es-elastic-user
+        extUseBasicAuth: true
+        extUseTls: true
```

Comment:
Haha, this is the case where you want to use the snake_case (it's in the Ansible now and the conversion has happened.

ext_host_url
ext_tls_server_name
etc...

Reply:
Awww, damnit! I'll need to switch the templates too. Gets me every time. You'd think I'd learn, but I guess not.

Reply:
It's definitely a tricky rule!